theme/layout/z-indexes.less

```diff
@@ -0,0 +1,5 @@
+@z-index-header-overlay: 4;
```

Comment:
Yeah, we can leave the values for now if you like, we can always revisit at a better time.